### ISSUE TITLE
ModuleNotFoundError in aef8ad4

### ISSUE THREAD

**Issue by simremy** [metgem/metgem#107]
Hi Nicolas, 

I couldn't run [aef8ad4](https://github.com/metgem/metgem/commit/aef8ad4924e84f07d26e65499c73b55a3f717b7f)
Here is traceback : 

Take care
Simon

`C:\Users\Simon\Documents\GitHub\metgem>conda activate MetGem

(MetGem) C:\Users\Simon\Documents\GitHub\metgem>python MetGem
Traceback (most recent call last):
  File "C:\Users\Simon\Documents\GitHub\metgem\MetGem", line 4, in <module>
    metgem_app.run()
  File "C:\Users\Simon\Documents\GitHub\metgem\metgem_app\metgem.py", line 142, in run
    importlib.import_module('.ui', 'metgem_app')
  File "C:\Users\Simon\.conda\envs\MetGem\lib\importlib\__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1030, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1007, in _find_and_load
  File "<frozen importlib._bootstrap>", line 986, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 680, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 850, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "C:\Users\Simon\Documents\GitHub\metgem\metgem_app\ui\__init__.py", line 1, in <module>
    from .main_window import MainWindow
  File "C:\Users\Simon\Documents\GitHub\metgem\metgem_app\ui\main_window.py", line 41, in <module>
    from ..ui import widgets
  File "C:\Users\Simon\Documents\GitHub\metgem\metgem_app\ui\widgets\__init__.py", line 2, in <module>
    from .metadata import (CsvDelimiterCombo,
  File "C:\Users\Simon\Documents\GitHub\metgem\metgem_app\ui\widgets\metadata\__init__.py", line 1, in <module>
    from .widgets import NodesWidget, EdgesWidget
  File "C:\Users\Simon\Documents\GitHub\metgem\metgem_app\ui\widgets\metadata\widgets.py", line 4, in <module>
    from .nodes_ui import Ui_NodesWidget
ModuleNotFoundError: No module named 'metgem_app.ui.widgets.metadata.nodes_ui'`

**Comment by n-elie** [comment]
Hi Simon,

If you run metgem directly from source code, you have to compile ui files by yourself by running this command: `inv uic`
Please be sure you have all the required libraries (requirements changed for the last commit): `conda env update -f environment.dev.yml` and `conda env update -f environment.yml`

Nicolas

**Comment by i-guigon** [comment]
Hi there.

I have the same problem with the version of code I've download on 2022/20/16 (2476f9780ea7af67492d71aac69c7f8224e10f4d).
First I got the `ModuleNotFoundError: No module named 'metgem_app.ui.widgets.metadata.nodes_ui' error`, and after I ran `inv uic` I got the following error:
`AttributeError: 'MainWindow' object has no attribute 'actionImportIdentityAnnotations'`.
I also ran the conda commands mentioned above.

Thank you for you help.
